journals/settings/base.py

```diff
@@ -76,6 +76,7 @@ def root(*x): return join(abspath(PROJECT_ROOT), *x)
 INSTALLED_APPS += WAGTAIL_APPS
 
 MIDDLEWARE_CLASSES = (
+    'journals.apps.core.middleware.SettingsOverrideMiddleware',
```

Comment:
do existing middleware classes still get called on requests as well? Assuming yes. Specifically want to make sure `wagtail.wagtailcore.middleware.SiteMiddleware'` gets called

Reply:
Yeah. It should effect other middlewares.